packages/web/src/features/translation/verse-utils.ts

```diff
@@ -30,6 +30,15 @@ export function verseCount(bookId: number, chapterNumber: number): number {
   return verseCounts[bookId - 1][chapterNumber - 1];
 }
 
+/**
+ * Check if the verse ID comes from an Old Testament book.
+ * @param verseId The verse ID to check, in the form BBCCCVVV.
+ * @returns true when the verse ID comes from an Old Testament book, false otherwise.
+ */
+export function isOldTestament(verseId: string): boolean {
+  return parseVerseId(verseId).bookId < 40;
```

Comment:
```suggestion
  const matthewBookId = 40;
  return parseVerseId(verseId).bookId < matthewBookId;
```

I was going to add an explanation comment, but then I realized a well named variable would make the magic number self-explanatory.